goinp/goinp_test.go

```diff
@@ -425,3 +428,227 @@ func TestSelectFromStringsFromReader(t *testing.T) {
 		require.EqualError(t, err, "invalid option: You entered a number greater than the last option's number")
 	}
 }
+
+func testAskOptionWrapper(title string, defaultValue string, optional bool, stdin string, options ...string) (string, string, error) {
+	in, err := ioutil.TempFile("/tmp", "gotest")
+	if err != nil {
+		return "", "", err
+	}
+	defer in.Close()
+
+	out, err := ioutil.TempFile("/tmp", "gotest")
```

Comment:
Could a bytes.NewReader(byteArray) be used as input and output stream?

Reply:
Sure

Reply:
Checked and it is not possible to create file descriptor in testing environment, so only files are possible